srcs/vm/vm.c

```diff
@@ -116,7 +115,7 @@ void        vm_start(t_arena *arena_s)
         if (DEBUG_MAX_CYCLES && arena_s->cycle_count > DEBUG_MAX_CYCLES)
             break;
 		if (arena_s->dump_flag != -1 &&
-		arena_s->dump_flag <= arena_s->cycle_count)
+		arena_s->dump_flag == arena_s->cycle_count)
```

Comment:
Moet wel <= zijn, bij -d 0 geeft originele corewar de dump en zonder <= doet de onze dat niet omdat cycle_count voor deze line al op 1 staat

Reply:
Dus dump 0 en dump 1 doen dan het zelfde?

Reply:
Dat ook, wat dan wel kan is
`arena_s->dump_flag == arena_s->cycle_count - 1`